vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -291,6 +291,9 @@ and
                 do! setupProjectsAfterSolutionOpen() 
             }
         setupProjectsAfterSolutionOpen() |> Async.StartImmediate
+
+        let theme = package.ComponentModel.DefaultExportProvider.GetExport<ISetThemeColors>().Value
```

Comment:
I guess GetExport returns a Lazy, and the Value will always be found because it is Exported, right here in this file.  So I answered my oen question :-)